components/layouts/AppLayout/index.js

```diff
@@ -0,0 +1,74 @@
+import { MenuOutlined } from "@ant-design/icons";
+import { Button, Drawer, Layout } from "antd";
+import { useEffect, useState } from "react";
+import AppMenu from "~/components/AppMenu";
+
+import styles from "./style.module.css";
+
+const { Content, Sider } = Layout;
+
+function AppLayout({ children, hidePrimaryMenu }) {
```

Comment:
```suggestion
function AppLayout({ children, hidePrimaryMenu = false}) {
```

Maybe we could add a default value to `hidePrimaryMenu` since it would only be `true` in very few cases.